### DISCUSSION TITLE
Estimating channels

### DISCUSSION THREAD

**Discussion by pfeatherstone** [jgaeddert/liquid-dsp#344] [Ideas]
Can liquid use something like [wiener filter](https://en.wikipedia.org/wiki/Wiener_filter) to estimate multi-path channels?
Cheers

**Comment by jgaeddert** [comment]
I suppose it could, but liquid doesn't have a wiener filter implemented. Probably the slowest piece would be inverting the Toeplitz Wiener-Hopf matrix system of equations as liquid doesn't yet take advantage of the symmetry in Toeplitz matrices.
